docs/options.md

```diff
@@ -550,7 +550,7 @@ a table of items, including arrays.
     single values.
```

Comment:
Not needed for this PR, but this is not true anymore, pip supports arrays, as long as we don't stop it in cibuildwheel (which I don't think we do), it should just work.

Reply:
Yes, pip support for multiple config settings with the same key was added in 23.1. Here's a link to the changelog: https://pip.pypa.io/en/stable/news/#v23-1

Reply:
That reminds me, this would still be true for 3.6.